hazelcast/hazelcast
# Test the Relational DB OOTB connector [HZ-1294]

This includes various test scenarios for the JDBC OOTB Connector (JDBC 
Mapping and `GenericMapStore`) such as:

- Test for various `IMap` methods that weren't already covered in
`GenericMapStoreIntegrationTest`
- Test behavior when underlying datastore is down/disconnected
- Verify JDBC mapping for use in SQL joins
- Check that client queries to JDBC return proper results even when
bouncing cluster members
